dmwmclient/datasvc.py

```diff
@@ -81,3 +81,199 @@ def __init__(self, client, datasvc_base=None, phedex_instance=None):
         )
         format_dates(df, ["replica.time_create", "replica.time_update"])
         return df
+
+    async def nodes(self, **params):
+
+        """Returns a simple dump of phedex nodes.
+        Parameters
+        ----------
+        node     PhEDex node names to filter on, can be multiple (*)
+        noempty  filter out nodes which do not host any data
+        """
+        resjson = await self.jsonmethod("nodes", **params)
+        df = pandas.io.json.json_normalize(
+            resjson["phedex"], record_path="node", record_prefix="node.",
+        )
+
+        return df
+
+    async def data(self, **params):
+
+        """Shows data which is registered (injected) to phedex
+        Parameters
+        ----------
+        dataset                  dataset name to output data for (wildcard support)
+        block                    block name to output data for (wildcard support)
+        file                     file name to output data for (wildcard support)
+        level                    display level, 'file' or 'block'. when level=block
+                                 no file details would be shown. Default is 'file'.
+                                 when level = 'block', return data of which blocks were created since this time;
+                                 when level = 'file', return data of which files were created since this time
+        create_since             when no parameters are given, default create_since is set to one day ago
+        """
+        resjson = await self.jsonmethod("data", **params)
+        out = []
+        for _instance in resjson["phedex"]["dbs"]:
+            for _dataset in _instance["dataset"]:
+                for _block in _dataset["block"]:
+                    for _file in _block["file"]:
+                        out.append(
+                            {
+                                "Dataset": _dataset["name"],
```

Comment:
I have a slight preference for keeping more machine-like names, i.e. `dataset_name` or `block_bytes` vs. the more human-readable version, as these routines are more likely to be used in scripts.  What if we have some optional flag in the function signature to return a more human-readable dataframe?

Reply:
The other consideration is that every departure from the naming conventions of the original datasvc output (and its associated documentation) provides another possible point of confusion.